### PR TITLE
Verify functionality with default storage class and update docs accordingly.

### PR DESCRIPTION
**Pull request by jhamacher-indeed**

### CODE REVIEW

File: docs/storage.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                - ReadWriteOnce
+              persistentVolumeReclaimPolicy: Retain
+              storageClassName: local-storage
+              hostPath:
+                path: "/data/volume-0003"
+            ---
+        </code>
+
+        <h2>Update RabbitMQ Spec</h2>
+        <p>Change the <code>storageClassName</code> in your RabbitMQ instance spec to reference <code>local-storage</code> rather than <code>rook-ceph-block</code>.</p>
```
Comment (bfink13) [inline docs/storage.html:86]: maybe add another short section about rook-ceph-block

File: docs/storage.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    </head>
+    <body>
+        <h1>Storage Examples</h1>
+        <h2>Overview</h2>
+        <p>This page provides examples of different storage configurations.</p>
+
+        <h1>Default Storage</h1>
+        <p>Removing the <code>storageClassName</code> field from the RabbitMQ instance spec will result in each pod attempting to claim a PersistentVolume with the default storage class.</p>
+
+        <p>For this to work seemlessly, you'll need:</p>
```
Comment (ghost) [inline docs/storage.html]: nit: seamlessly

Comment (jhamacher-indeed) [reply docs/storage.html]: Nice catch.  The older I get the worse I am for things like that.

File: docs/storage.html (hunk 1 above)
Comment (jhamacher-indeed) [reply docs/storage.html:86]: I added a bit; I'm not sure how much detail you wanted so let me know if there's something in particular I should address.

**Review by bfink13 — APPROVED** [review]

**Pull request merged**